src/modality-specific-files/microelectrode-electrophysiology.md

```diff
@@ -0,0 +1,1017 @@
+-   Range: -180° to +180° (or 0° to 360°)
+-   Positive rotation is clockwise when viewed from above
+
+!!! note "Source Attribution"
+
+    The coordinate system conventions and angle definitions presented in this section are adapted from the [BrainSTEM documentation](https://support.brainstem.org/datamodel/schemas/coordinates/).
+
+## Coordinate System JSON (`*_coordsystem.json`)
+
+<!--
+This block generates a filename templates.
+The inputs for this macro can be found in the directory
+  src/schema/rules/files/raw
+and a guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_filename_template("raw", datatypes=["icephys", "ecephys"], suffixes=["coordsystem"]) }}
+
+This `*_coordsystem.json` file contains the coordinate system in which electrode
+positions are expressed. The associated MRI, CT, X-Ray, or operative photo can
+also be specified.
+
+This file is **OPTIONAL** when using default probe-relative coordinates (no `space-<label>` entity).
+This file is **REQUIRED** when the [`space-<label>`](../appendices/entities.md#space) entity is used
+in the filename to specify electrode positions in an anatomical or stereotaxic coordinate system.
+When a `*_space-<label>_coordsystem.json` file is present, the corresponding `*_space-<label>_electrodes.tsv`
+file with the same space label MUST also be present.
+
+General fields:
+
+<!--
+This block generates a metadata table.
+These tables are defined in
+  src/schema/rules/sidecars
+The definitions of the fields specified in these tables may be found in
+  src/schema/objects/metadata.yaml
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_json_table("json.microephys.microephysCoordsystemGeneral") }}
+
+Fields relating to the microelectrode electrophysiology electrode positions:
+
+<!--
+This block generates a metadata table.
+These tables are defined in
+  src/schema/rules/sidecars
+The definitions of the fields specified in these tables may be found in
+  src/schema/objects/metadata.yaml
+A guide for using macros can be found at
+ https://github.com/bids-standard/bids-specification/blob/master/macros_doc.md
+-->
+{{ MACROS___make_json_table("json.microephys.microephysCoordsystemPositions") }}
+
+`*_coordsystem.json` files SHOULD NOT be duplicated for each data file,
+for example, across multiple tasks.
+The [inheritance principle](../common-principles.md#the-inheritance-principle) MUST
+be used to find the appropriate coordinate system description for a given data file.
+If electrodes are repositioned, it is RECOMMENDED to use multiple sessions to indicate this.
+
+### Default probe-relative coordinate systems
+
+When no [`space-<label>`](../appendices/entities.md#space) entity is specified in the filename,
+electrode positions in `*_electrodes.tsv` are **probe-relative coordinates**:
```

Comment:
chatting with @CodyCBakerPhD who raises a good point (I think) that if we are to describe probe geometries , either standard, or custom under `/probes`, then why do we want this yet another way to describe coordinates for the electrodes on the probes, and in the `_probes.tsv` table in particular?  they would not be different for the same probe manufacturer/model, so why to repeat across the files? if very custom, a file under `/probes` could still be used to describe it.